src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,28 @@
 package mate.academy;
 
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private final ConcurrentHashMap<EventListener, Boolean> listeners = new ConcurrentHashMap<>();
```

Comment:
Using a ConcurrentHashMap with a Boolean value seems unnecessary if the value is always 'true' and never used. If you only need a set of listeners, consider using a ConcurrentHashSet (which can be emulated using ConcurrentHashMap.newKeySet()) to avoid storing redundant values.